x-pack/plugins/ingest_manager/server/services/epm/packages/install.ts

```diff
@@ -15,6 +15,17 @@ import { installTemplates } from '../elasticsearch/template/install';
 import { installPipelines } from '../elasticsearch/ingest_pipeline/install';
 import { installILMPolicy } from '../elasticsearch/ilm/install';
 
+export async function installLatestPackage(options: {
```

Comment:
This function is going to be called every time someone open ingest manager do we want to install packages only they are not already installed?

Reply:
@nchaulet yea, just addressed this in a comment above.  i'm not sure if we want to always check for the latest version and update it

Reply:
@nchaulet added a new function that checks first